VulnerableApplication/Global.asax.cs

```diff
@@ -17,5 +17,7 @@ protected void Application_Start()
             RouteConfig.RegisterRoutes(RouteTable.Routes);
             BundleConfig.RegisterBundles(BundleTable.Bundles);
         }
+        
+        public string ChangePassword { get; set; }
```

Comment:
3 more lines of code need to be covered by tests to reach the minimum threshold of 65.0% lines coverage.